Rakefile

```diff
@@ -432,11 +432,6 @@ end
 def update_measures
   require 'openstudio'
 
-  # Apply rubocop
-  command = "rubocop --auto-correct --format simple --only Layout"
-  puts "Applying rubocop style to measures..."
-  system(command)
```

Comment:
I'm confused. Are you expecting someone to manually execute this rubocop command?

Reply:
No. The issue is that nothing was requiring that `update_measures` be run before committing.